code/modules/research/xenoarchaeology/artifact/artifact_unknown.dm

```diff
@@ -83,7 +85,7 @@ var/list/valid_secondary_effect_types = list(\
 		src.investigation_log(I_ARTIFACT, "|| spawned with a secondary effect [secondary_effect.artifact_id]: [secondary_effect] || range: [secondary_effect.effectrange] || charge time: [secondary_effect.chargelevelmax] || trigger: [secondary_effect.trigger].")
 		if(prob(75))
 			src.investigation_log(I_ARTIFACT, "|| secondary effect [secondary_effect.artifact_id] starts triggered by default.")
-			secondary_effect.ToggleActivate(0)
+			secondary_effect.ToggleActivate(2)
```

Comment:
That argument smells like a magic number.  Given that the current proc signature is `/datum/artifact_effect/proc/ToggleActivate(var/reveal_toggle = 1)`, this should probably be made a `#define` so it's clearer what the difference between 0 and 2 are.